mpas_analysis/ocean/streamfunction_moc.py

```diff
@@ -353,22 +349,16 @@ def _compute_moc_climo_postprocess(self):  # {{{
         if not os.path.exists(outputFileClimo):
             print '   Load data...'
 
-            cachePrefix = '{}/meanVelocity'.format(outputDirectory)
-
-            if self.startYearClimo == self.endYearClimo:
-                yearString = '{:04d}'.format(self.startYearClimo)
-                velClimoFile = '{}_year{}.nc'.format(cachePrefix, yearString)
-            else:
-                yearString = '{:04d}-{:04d}'.format(self.startYearClimo,
-                                                    self.endYearClimo)
-                velClimoFile = '{}_years{}.nc'.format(cachePrefix, yearString)
-
-            annualClimatology = xr.open_dataset(velClimoFile)
+            annualClimatology = xr.open_dataset(self.velClimoFile)
+            # rename some variables for convenience
+            annualClimatology = annualClimatology.rename(
+                    {'timeMonthly_avg_normalVelocity': 'avgNormalVelocity',
+                     'timeMonthly_avg_vertVelocityTop': 'avgVertVelocityTop'})
```

Comment:
This is where we could open the `annualClimatology` with the `generalized_reader` instead of just `xr.open_dataset` if we wanted to use the `variableMap` (and maintain backwards compatibility).  It would be total overkill because we are only opening one file and don't need any other capabilities of the generalized reader aside from the variable map but I doubt there would be too much overhead...

The same approach might work for other climatologies as we move to `ncclimo`.

I'm more in favor of letting go of backward compatibility if we can.  The variable map is confusing for developers.  But you have a better sense than I do if it's necessary.